### ISSUE TITLE
Declaration cart page

### ISSUE THREAD

**Issue by SaOgaz-NOAA** [nwfsc-fram/boatnet#1215]
- [x] Hook up Submit Declarations button
- [x] Add x's to the dec cards so they  can be removed from cart
- [x] add back button
- [x] your cart is empty message (see add declaration button etc)
- [x] fill in dec card details appropriately
- [x]  make a decision about observer status
- [ ] bug with cart not appropriately reloading when first exit dec worksheet page

Note to make a new issue: handle when can't upload submit new declarations back to couch

**Comment by SaOgaz-NOAA** [comment]
I'm waiting on Seth to be back to help with my loading bug, pretty sure he'll know how to fix this pretty easily. So moving that to new ticket and closing this one!
